.github/workflows/build_and_test_debug.yml

```diff
@@ -132,6 +132,29 @@ jobs:
 
       - name: Build iOS Client
         run: npm run action cordova/build ios
+        
+  spm_debug_build:
+    name: Swift Package Debug Build
+    runs-on: macos-12
+    timeout-minutes: 30
+    strategy:
```

Comment:
I'm not sure you need a matrix here, since you are already listing the commands for each platform below.